pom.xml

```diff
@@ -18,51 +18,51 @@
   <modules>
     <module>bom</module>
     <module>parent</module>
-    <module>atomix</module>
-    <module>auth</module>
-    <module>broker</module>
-    <module>broker-client</module>
-    <module>protocol-test-util</module>
     <module>dist</module>
-    <module>protocol</module>
-    <module>bpmn-model</module>
-    <module>logstreams</module>
-    <module>stream-platform</module>
-    <module>engine</module>
-    <module>transport</module>
-    <module>util</module>
-    <module>test-util</module>
-    <module>msgpack-core</module>
-    <module>msgpack-value</module>
     <module>build-tools</module>
-    <module>gateway-protocol</module>
-    <module>gateway-protocol-impl</module>
-    <module>gateway</module>
-    <module>gateway-rest</module>
-    <module>clients/java</module>
```

Comment:
🔧 I already shared it earlier, but I feel like the java client is outside of Zeebe's scope and may benefit from existing in its own clients module

Reply:
Thanks for bringing this up. As I mentioned in the Team sync, right now these clients are only targeting the Zeebe grpc API, so they relate to Zeebe. There is a plan to create a common API. I think then yes it makes sense to move them but right now not. 

Likely we might need to move it again, let us see.